### ISSUE TITLE
Task: Create a Splitter Method

### ISSUE THREAD

**Issue by Sam54000** [Sam54000/bids_explorer#22]
### Description

In order to be used in ML engineering, the dataset should be able to be splitted. The idea is to develop a split method that would return a tuple of architecture objects (The main database but splitted). The arguments of the method would accept either a float between 0 and 1 that would define the percentage of data (0.5 would give a 50% split, 0.25 would give one object with 25% of the data and another object with 75% of the data). The argument would also accept one of the following [scikit-learn model selection objects](https://scikit-learn.org/stable/api/sklearn.model_selection.html):
- [GroupKFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GroupKFold.html#groupkfold)
- [GroupShuffleSplit](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GroupShuffleSplit.html)
- [KFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.KFold.html)
- [LeaveOneGroupOut](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.LeaveOneGroupOut.html#leaveonegroupout)
- [LeaveOneOut](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.LeaveOneOut.html)
- [LeavePGroupsOut](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.LeavePGroupsOut.html)
- [LeavePOut](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.LeavePOut.html)
- [RepeatedKFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RepeatedKFold.html)
- [RepeatedStratifiedKFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RepeatedStratifiedKFold.html)
- [ShuffleSplit](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.ShuffleSplit.html)
- [StratifiedGroupKFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.StratifiedGroupKFold.html)
- [StratifiedKFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.StratifiedKFold.html)
- [StratifiedShuffleSplit](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.StratifiedShuffleSplit.html)

When a scikit-learn model_selection object is provided, the split method will internally call the model_selection object's split method by providing the indices of the `architecture.database (pandas.DataFrame)` as argument (`numpy.ndarray`).
Other optional argument for higher granularity can be provided such as subjects, sessions and other.

### Tasks

- [ ] Task 1
